rdr_service/offline/study_nph_biobank_import_inventory_file.py

```diff
@@ -81,8 +81,10 @@ def _convert_csv_row_to_stored_sample_object(csv_obj: Dict[str, Union[str, int]]
         "lims_id": csv_obj["LIMS_SAMPLE_ID"],
         "status": stored_sample_status.get(csv_obj["STATUS"]),
         "disposition": stored_sample_status.get(csv_obj["DISPOSITION"]),
-        "specimen_volume_ul": csv_obj.get("SPECIMEN_VOLUME_UL"),
-        "freeze_thaw_count": csv_obj.get("FREEZE_THAW_COUNT")
+        "specimen_volume_ul":
+            csv_obj.get("SPECIMEN_VOLUME_UL") if csv_obj.get("SPECIMEN_VOLUME_UL") != '' else None,
+        "freeze_thaw_count":
+            csv_obj.get("FREEZE_THAW_COUNT") if csv_obj.get("FREEZE_THAW_COUNT") != '' else None
```

Comment:
Instead of the ternary, you can do `csv_obj.get("SPECIMEN_VOLUME_UL",  None)`

Reply:
The problem is that it is finding a value, but that value is an empty string. the default return of .get is None according to docs.

Reply:
Did you try this in a test??
```
Python 3.11.4 (v3.11.4:d2340ef257, Jun  6 2023, 19:15:51) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> ff ={}
>>> ff['darryl'] = ''
>>> if ff.get('darryl'):
...     print('Hello')
...
>>>
```